uber/templates/dept_checklist/overview.html

```diff
@@ -91,7 +91,7 @@
 
 
 {% for dept, dept_name, heads in overview %}
-    <tr><td colspan="checklist|length|add:1" class="dept">{{ dept_name }}</td></tr>
+    <tr><td {# **even though this breaks the table, it still looks better** colspan="{{ checklist|length + 1 }}" #} class="dept">{{ dept_name }}</td></tr>
```

Comment:
i.... me? did i do this? this looks like something i would do.

probably saying that our original code generated bad HTML but this bad HTML looks ok.  

in a future PR, whatever poor soul has to rip out some of our HTML/CSS code can throw us over the coals for this then :)    but for now, that's a problem for future us'es :)